src/NServiceBus.Core/Pipeline/satellitePipelineExecutor.cs

```diff
@@ -17,7 +14,12 @@ public SatellitePipelineExecutor(IBuilder builder, SatelliteDefinition definitio
 
         public Task Invoke(MessageContext messageContext)
         {
+            messageContext.Context.Set(messageContext.TransportTransaction);
```

Comment:
so this is set to avoid updating downstreams to use `messageContext.TransportTransaction` instead, did I get that right?